compiler/src/dotty/tools/dotc/reporting/Message.scala

```diff
@@ -176,8 +175,8 @@ object Message:
       def needsExplanation(entry: Recorded) = entry match {
         case param: TypeParamRef => ctx.typerState.constraint.contains(param)
         case param: ParamRef     => false
-        case skolem: SkolemType => true
-        case sym: Symbol =>
+        case skolem: SkolemType  => true
+        case sym: Symbol         =>
           ctx.gadt.contains(sym) && ctx.gadt.fullBounds(sym) != TypeBounds.empty
```

Comment:
This equality check looks like the previous tweak to use `=:=` for this comparison. I have not yet investigated / learned about it.